public/index.html

```diff
@@ -18,16 +30,22 @@ <h1>Ellevest Take Home</h1>
           </p>
           <p>
-            This exercise uses an API, please review the documentation <a href="api-docs.html" target="_blank">here</a>. The base URL for the API is the URL for this page (ie. if the URL for this page is <code>https://ellevest-take-home.glitch.me</code> the Transfers API endpoint would be <code>https://ellevest-take-home.glitch.me/transfers</code>).
+            This exercise uses an API, please review the documentation <a href="api-docs.html" target="_blank">here</a>. The base URL for the API is <code class="baseUrl"></code>. As an example, the Transfers API endpoint would be <code><span class="baseUrl"></span>/transfers</code>.
           </p>
           <p>
             You may use the language of your choice for this exercise and include any additional libraries you require to complete the task. Your implementation should have valid syntax and be excutable. No need to spend more than two hours working on the exercise – it's alright if you have not completed both parts within that time frame. When you're done, please email <a href="mailto:people@ellevest.com">people@ellevest.com</a> with your attached solution. Please reach out using the same email address if you have any questions or concerns.
           </p>
+          <p>
+            Here at Ellevest we <i>love</i> testable, modular, and clean code. Please treat this exercise as if it's production code and part of a larger system and consider how your code would fit into the broader context of an application.
+          </p>
+          <p>
+            Be prepared to discuss the trade-offs you made in your implementation and to participate in a peer code-review discussion.
+          </p>
         </div>
         <div class="row">
           <h2>Part I</h2>
           <p>
-            Using the API, implement a method to build the data for a "social" transfers feed, combining information from whichever API resources are necessary. Transfers with a "failed" status should be excluded from the feed. Your method should return an Array of Objects (Hashes, Dictionaries, etc.) with the following information:
+            Using the API, implement a method to build the data for a "social" transfers feed, combining information from whichever API resources are necessary. Transfers with a "failed" status should be excluded from the feed. Your method should return an Array of Objects (Hashes, Dictionaries, etc.). Find an example output below:
```

Comment:
I think this change is confusing, because then we're not ever saying what the expected output is, just "example" output, which could be interpreted as the fields themselves just being an example. I think "with the following information" was fine, I haven't seen anyone confused about the requirements here.